moped-editor/src/views/projects/projectView/ProjectComponentEdit.js

```diff
@@ -59,6 +59,7 @@ const useStyles = makeStyles(theme => ({
  * @param {function} handleCancelEdit - The function to call if we need to cancel editing
  * @param {function} projectRefetchFeatures - Reload parent component's features
  * @param {Object} projectFeatureCollection - The entire project's feature collection GeoJSON (optional)
+ * @param {function} projectSetFeatureCollection - Allows us to clear the feature collection for reloading
```

Comment:
is this defined here in ProjectComponentEdit?

Reply:
@chiaberry 🙏 thank you so much, I totally missed that and the fact we do not directly use projectRefetchFeatures anymore. Just got that cleaned up.